.travis.yml

```diff
@@ -27,6 +31,7 @@ stages:
 jobs:
   include:
     - python: "2.7"
+      env: PY=2
```

Comment:
Why /how this variable is used ?

Reply:
You can run make targets with PY=2 to test in or install a python 2 environment:

      make PY=2  install test

If PY= is unset, PY=3 is the default.  To see more options run help

     make help

Reply:
The PY=2 condition in main Makefile was only a needed addition to skip the pylint test. Mainly the PY variable sets a py2 or py3 environment.